src/main/java/dev/meyi/bn/handlers/MouseHandler.java

```diff
@@ -17,11 +17,15 @@ public class MouseHandler {
   public void mouseActionCheck(TickEvent e) {
     if (e.phase == TickEvent.Phase.START) {
       if (BazaarNotifier.inBazaar) {
-        BazaarNotifier.modules.movementCheck();
+        if(BazaarNotifier.config.legacyMovement) {

```

Comment:
legacyMovement? 

Is this referring to moving it through the config as the new method and moving it through dragging in the menu as "legacy"? Just checking

Reply:
Yes, this allows dragging and rescaling in the menus